src/components/checkbox/_type/checkbox_type_button.bemhtml

```diff
@@ -0,0 +1,27 @@
+block('checkbox').mod('type', 'button')(
+    content()(function() {
+        const React = require('react');
+        const Button = require('../src/components/button/button');
+
+        const ctx = this.ctx,
+            mods = this.mods;
+
+        const originalContent = applyNext();
```

Comment:
This [temlate](https://github.com/awinogradov/bem-components/blob/v2/common.blocks/checkbox/_type/checkbox_type_button.bemhtml#L6) says yes